packages/spatial/src/physics/components/ColliderComponent.tsx

```diff
@@ -75,7 +88,25 @@ export const ColliderComponent = defineComponent({
         Physics.removeCollider(physicsWorld, entity)
         hasCollider.set(false)
       }
-    }, [physicsWorld, component.shape, !!rigidbodyComponent?.initialized?.value, transform.scale])
+    }, [
+      physicsWorld,
+      component.shape,
+      !!rigidbodyComponent?.initialized?.value,
+      transform.scale,
+      component.centerOffset,
+      component.boxSize,
+      component.radius,
+      component.height,
+      meshComponent
+    ])
+
+    useEffect(() => {
+      if (!physicsWorld) return
```

Comment:
this should be inside the return function

Reply:
fixing